report/test/clients/bundle-test.js

```diff
@@ -26,12 +26,17 @@ describe('lighthouseRenderer bundle', () => {
     document = window.document;
 
     global.window = global.self = window;
+    global.window.requestAnimationFrame = fn => fn();
```

Comment:
Can we use the JSDOM implementation?

Reply:
they don't have one. :) https://github.com/jsdom/jsdom/issues/1963